### PR TITLE
(Re)organized the FusionMiddleware Test classes in Integration Suite for a fmw certification effort. 

### PR DESCRIPTION
**Pull request by anpanigr**
Purpose:  Reorganized the FusionMiddleware Test classes in Integration Suite for a fmw certification effort. 

(a) All the fmw tests are renamed to a class containing string Fmw
       ItJrfDomainInPV.java --> ItFmwDomainInPV.java 
       ItJrfMiiDomain.java --> ItFmwMiiDomain.java
       ItSamples.java --> ItWlsSamples.java  
      ItManageNs.java --> ItManageNameSpace.java
     
(b) Renamed the test methods  from  jrf-->fmw 

https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/4056/
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/4073/
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/4089/

### CODE REVIEW

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiFmwSample.java [hunk 1]
```diff
@@ -47,21 +47,19 @@
 import static org.junit.jupiter.api.Assertions.assertTrue;
```
Comment (tbarnes-us) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiFmwSample.java:47]: There is significant complex duplicate code between ItMIIFmwSample and ItMIIWlsSample, including variable declaration, initialization, and helper methods.

Comment (tbarnes-us) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiFmwSample.java:47]: Can the common information in the two classes be moved to a shared parent class?

Comment (anpanigr) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiFmwSample.java]: I removed the two classes ITMiiFmwSample and ItMiiWlsSample and restored the ItMiiSample.  I can skip WLS tests in FMW image and vice versa using environment variable  SKIP_WLS(JRF)_SAMPLES

File: integration-tests/pom.xml [hunk 2]
```diff
@@ -238,21 +238,50 @@
...
+                  **/ItFmwSample.java
+                </includes-failsafe>
             </properties>
         </profile>
         <profile>
             <id>oke-cert</id>
             <properties>
                 <skipITs>false</skipITs>
-                <includes-failsafe>**/ItParameterizedDomain.java,**/ItServerStartPolicy.java,**/ItMiiUpdateDomainConfig.java,**/ItIntrospectVersion.java,**/ItJrfDomainInPV.java,
+                <includes-failsafe>**/ItParameterizedDomain.java,**/ItServerStartPolicy.java,**/ItMiiUpdateDomainConfig.java,**/ItIntrospectVersion.java,**/ItFmwDomainInPV.java,
```
Comment (vanajamukkara) [inline integration-tests/pom.xml:284]: can u put this also one class on each line like u did above

**Review by vanajamukkara — APPROVED** [review]
LG, added a minor comment

File: integration-tests/pom.xml (hunk 2 above)
Comment (anpanigr) [reply integration-tests/pom.xml]: We do not need java extension.  I updated all the profile with multi line class inclusion without java extension.

**Review by maggiehe00 — APPROVED** [review]
LGTM

**Pull request merged**